### DISCUSSION TITLE
Should we use fosshost to test curl on ARM?

### DISCUSSION THREAD

**Discussion by pheiduck** [curl/curl#8640] [Ideas]
As Circle CI removed the arm hosts. We need a new CI to test cURL on ARM.
I think discussions is a better place to "discuss" about this topic.
https://github.com/curl/curl/pull/8630#issuecomment-1077675400

**Comment by bagder** [comment]
Sure, why not? If we can add jobs there that contribute to the total CI set and makes a better curl.
